docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    =
```

Comment:
Nice :) It's been a long time for me to work on Sphinx. Happy to see these changes.